recipes/py2dm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "py2dm" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py2dm-{{ version }}.tar.gz
+  sha256: 447e3af1e5adfc7a898acd5e5744ab58f86d995cf615147629fec7b4bf605be7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    # Without these the compilation of the C++ extension fails on Python 3.7
+    - typing-extensions   # [py<38]
+    - typing_extensions   # [py<38]
+    - cached-property     # [py<38]
```

Comment:
These are definitely not needed in build. If removing them causes an error then it's a sign of a different problem in the recipe.

Reply:
Okay, I will investigate further.